imagery/i.segment/i.segment.html

```diff
@@ -277,6 +275,9 @@ <h2>REFERENCES</h2>
 <h2>SEE ALSO</h2>
 
 <em>
+<a href="i.segment.stats.html">i.segment.stats</a>
+<a href="i.segment.uspo.html">i.segment.uspo</a>
+<a href="i.segment.hierarchical.html">i.segment.hierarchical</a>
```

Comment:
```suggestion
<a href="https://grass.osgeo.org/grass8/manuals/addons/i.segment.stats.html">i.segment.stats</a> (addon),
<a href="https://grass.osgeo.org/grass8/manuals/addons/i.segment.uspo.html">i.segment.uspo</a> (addon),
<a href="https://grass.osgeo.org/grass8/manuals/addons/i.segment.hierarchical.html">i.segment.hierarchical</a> (addon)
```

Reply:
grass8 is an known alias?

Reply:
Yes, server side a redirect to the latest G8 stable.